app/views/orgs/_departments.html.erb

```diff
@@ -0,0 +1,48 @@
+<%# locals: { context, title, tooltip } %>
+<% department =  department.nil? ? Department.new :  department %>
+<%= form_for(department, url: "/departments/create_or_update", html: { multipart: true, method: method, id: "edit_org_departments_form" } ) do |f| %>
```

Comment:
use the rails route helpers instead of the hard coded path. Do a `rake routes` and then find the line with `departments#create_or_update`. You should then see something like `POST department` at the beginning of that line. Take that value and add 'path' to the end for the route helper (e.g. `department_path`)